container/example-config/basic.toml

```diff
@@ -0,0 +1,27 @@
+[[agent]]
+id = "test agent"
+name = "Holo Tester"
+key_file = "holo_tester.key"
+
+[[dna]]
+id = "app spec rust"
+file = "app-spec-rust.hcpkg"
+hash = "Qm328wyq38924y"
+
+[[instance]]
+id = "app spec instance"
```

Comment:
We had also talked about just using (agent_key, dna_hash) as the unique id for instances. If this separate notion of id is not implemented, we could take it out...

Reply:
I intend to fill the id this way when the config gets created from code (`install_app()`), but I like having an explicit id in the config file especially if a human writes it manually.